tests/integration-tests/configManager/configHandler-test.js

```diff
@@ -71,11 +73,15 @@ const requestBodyNotValidRequire = {
     }
 };
 
-before(async () => {
-    await configRequestCreator.init();
-});
-
 describe('update and get config', () => {
```

Comment:
const requestBody =

put unique value for each config. (with the key as prefix)